README.md

```diff
@@ -102,6 +102,10 @@ The following variable defaults are defined in `defaults/main.yml`.
 
     list of IP addresses to ban (default: `[]`)
 
+* `minecraft_server_properties`
+
+    list of server.properties entries (e.g. `server-port=25565`) to set (default: `[]`)
```

Comment:
All of the [properties](http://minecraft.gamepedia.com/Server.properties#Minecraft_server_properties) are strictly key–value with simple types (boolean, string, integer). A dictionary seems a better fit than a list.